MoltenVK/MoltenVK/GPUObjects/MVKQueryPool.mm

```diff
@@ -281,12 +290,9 @@
 }
 
 void MVKOcclusionQueryPool::beginQueryAddedTo(uint32_t query, MVKCommandBuffer* cmdBuffer) {
+	// In multiview passes, one query is used for each view.
+	NSUInteger queryCount = cmdBuffer->getViewCount();
```

Comment:
Wait, what happens when we're *not* in a secondary command buffer?

Reply:
It would default to 1. This was the same as existing behavior, since the existing `_lastMultiviewSubpass` was never set (`recordBeginRenderPass` was never called).

The `queryCount` value here is only used as a sanity check on multipass queries that approach the end of the visibility buffer, so I toyed with simply removing all this code.

However, in the end, I added a `MVKCurrentSubpassInfo` helper in `MVKCommandBuffer` to track the current renderpass, subpass, or dynamic rendering, to derive a `viewMask` for the command corresponding to the currently added subpass, replacing and expanding on what `_lastMultiviewSubpass` was supposed to do.

So, even though it's just doing an edge-case validation test, we've got some plumbing to track other subpass command info if we need to.